src/hotspot/os/linux/malloctrace/mallocTrace.cpp

```diff
@@ -29,6 +29,7 @@
 #include "malloctrace/assertHandling.hpp"
 #include "malloctrace/locker.hpp"
 #include "malloctrace/mallocTrace.hpp"
+#include "malloctrace/mallocTrace2.hpp"
```

Comment:
Please annotate

Reply:
Is this needed in SapMachine-only code?

Reply:
you're right. Forget this comment then. 😄